backend/src/settings.rs

```diff
@@ -33,6 +36,11 @@ pub struct DatabaseSettings {
     pub cacertpath: Option<String>,
 }
 
+#[derive(Debug, Deserialize, Clone)]
+pub struct FcmSettings {
+    pub path: String,
```

Comment:
We could use `PathBuf` instead of `String` here. Could you please add a documentation comment, that this is a path to a service account key file in json format. Thanks!